files/build_templates/init_cfg.json.j2

```diff
@@ -83,6 +83,20 @@
             "state" : "disabled", {% endif %}
             "rate_limit_interval" : "600"
         }{%if not loop.last %},{% endif -%}
-{% endfor %} 
+{% endfor %}
+    },
+    "PASSW_HARDENING": {
+        "POLICIES":{
+            "state": "disabled",
```

Comment:
Since it is disabled by default I am trying to understand why do we need to explicitly add it in init_cfg.json?

Reply:
the feature is enabled in compilation time, and this is the "default" value to disable the feature in runtime.